From 9e379427c5b8ac595c2224d362ff4be089329d67 Mon Sep 17 00:00:00 2001
From: Eliad Peller <eliad@wizery.com>
Date: Sun, 22 May 2011 13:24:01 +0300
Subject: [PATCH 51/75] fix merge error (wl1271_cmd_set_peer_state)

wl1271_cmd_set_peer_state() call is being added to the wrong function
because the patch blob is not unique. fix it.
(this patch needs to be squashed finally, but currently leave it as
this merge error occurs on every rebase)
---
 drivers/net/wireless/wl12xx/main.c |    8 ++++----
 1 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/net/wireless/wl12xx/main.c b/drivers/net/wireless/wl12xx/main.c
index 8c75a75..b5ab0b3 100644
--- a/drivers/net/wireless/wl12xx/main.c
+++ b/drivers/net/wireless/wl12xx/main.c
@@ -3378,10 +3378,6 @@ static u64 wl1271_op_get_tsf(struct ieee80211_hw *hw)
 	if (ret < 0)
 		goto out_sleep;
 
-	ret = wl1271_cmd_set_peer_state(wl, hlid);
-	if (ret < 0)
-		goto out_sleep;
-
 out_sleep:
 	wl1271_ps_elp_sleep(wl);
 
@@ -3463,6 +3459,10 @@ static int wl1271_op_sta_add(struct ieee80211_hw *hw,
 	if (ret < 0)
 		goto out;
 
+	ret = wl1271_cmd_set_peer_state(wl, hlid);
+	if (ret < 0)
+		goto out_sleep;
+
 	ret = wl1271_ps_elp_wakeup(wl);
 	if (ret < 0)
 		goto out_free_sta;
-- 
1.7.0.4

